crates/search/src/aggregation.rs

```diff
@@ -46,6 +51,13 @@ pub enum Error {
     DatafusionError {
         source: datafusion::error::DataFusionError,
     },
+
+    #[snafu(display(
+        "The resulting aggregation result is inconsistent, which is an unexpected error. Error: {source}. Report a bug on GitHub: https://github.com/spiceai/spiceai/issues"
```

Comment:
```suggestion
        "The resulting aggregation result is inconsistent, which is an unexpected error.\n{source}\nReport a bug on GitHub: https://github.com/spiceai/spiceai/issues"
```

Error handling guide denotes the source is displayed on a new line.